caip2.md

```diff
@@ -0,0 +1,141 @@
+---
+namespace-identifier: icp-caip2
+title: ICP Namespace - Chains
+author: Quint Daenen (@q-uint)
+discussions-to: TODO
+status: Draft
+type: Standard
+created: 2023-05-17
+requires: CAIP-2
+---
+
+# CAIP-2
+
+*For context, see the [CAIP-2][] specification.*
+
+## Rationale
+
+The CAIP-2 namespace for the Internet Computer Protocol (ICP) provides a standardized system for identifying assets
+within the network. It promotes interoperability, efficient querying, and clarity in asset identification, benefiting
```

Comment:
What's the model for identifying resources? Do we want the addresses to be valid [URIs](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) or [IPFS paths](https://docs.ipfs.tech/how-to/address-ipfs-on-web/#dweb-addressing-in-brief)?

Reply:
[CAIP 19](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-19.md) describes resources/assets.
The scope of this document is only to define the `chainId`.